scripts/run_docker_build.sh

```diff
@@ -8,6 +8,7 @@ IMAGE_NAME="condaforge/linux-anvil"
 config=$(cat <<CONDARC
 
 channels:
+ - conda-forge/label/test
```

Comment:
Sorry about that... I feel bad since I am using your PR in my internal build for moths now and it is awesome!